forge-gui/res/cardsfolder/upcoming/ravenous_necrotitan.txt

```diff
@@ -2,9 +2,8 @@ Name:Ravenous Necrotitan
 ManaCost:2 B B
 Types:Creature Phyrexian Horror
 PT:6/6
-T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigSacrifice | TriggerDescription$ When CARDNAME enters the battlefield, sacrifice a land. Then conjure two cards named Mishra's Foundry onto the battlefield tapped.
-SVar:TrigSacrifice:DB$ Sacrifice | SacValid$ Creature | Defined$ You | ConditionCheckSVar$ X | ConditionSVarCompare$ LE3
-Corrupted — When CARDNAME enters the battlefield, sacrifice a creature unless an opponent has three or more poison counters.
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | CheckSVar$ X | SVarCompare$ LT3 | Execute$ TrigSacrifice | TriggerDescription$ Corrupted — When CARDNAME enters the battlefield, sacrifice a creature unless an opponent has three or more poison counters.
```

Comment:
@tool4ever i need to ask Judges, does the ability trigger when Corrupted isn't active

but only is active when the ability resolves?

Reply:
so it should not be a trigger condition, but a condition on the spellAbility

Reply:
yea that shouldn't have been changed

Reply:
@Simisays ^